clubs_guides/urls.py

```diff
@@ -1,3 +1,18 @@
 from django.conf.urls import url
 
-# Create you url mappings here.
+from clubs_guides.views import (
+    ClubsGuidesListView,
+    ClubsGuideView,
+    CategoryListView,
+)
+
+
+urlpatterns = [
+    url("^$", ClubsGuidesListView.as_view(), name="clubsguide-list"),
+    url(r"^(?P<pk>[0-9]+)", ClubsGuideView.as_view(), name="clubsguide"),
+    url(
```

Comment:
I'm familiar with pep 8... I have opinions on some of the calls in it, but "sticking to pep 8" is better than "sticking with some custom ruleset" when it comes to python.